workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
 set -e
+oc adm wait-for-stable-cluster --minimum-stable-period=1m --timeout=5m
```

Comment:
Just curious if we need this here, while we already have cluster health [check enabled](https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/ocp.go#L53) by default in ocp-wrapper driver code.